labs/a7.md

```diff
@@ -231,9 +240,14 @@ Then, create a file `/etc/bind/db.example.com` to contain the responses to give
 if anyone sends requests to your DNS server for `example.com`. The easiest way
 to do this is generally to copy an existing config and then make changes from
 there to get what you want for your config instead of having to start from
-scratch. To make this easier, we've provided a valid config at
-`/opt/lab7/db.example.com` that you can copy in place at
-`/etc/bind/db.example.com`. It is prefilled with your VM's IP, and includes a
+scratch. 
+
+To make this easier, we've provided a valid config in 
+[decal-labs](https://github.com/0xcf/decal-labs/blob/master/a7/db.example.com)
+that you can copy in place at
+`/etc/bind/db.example.com`. **You'll need to edit the config to include your VM's IP address!**
```

Comment:
There's a couple things to edit, one with the IP, and another with a domain name